src/quest_manager/models.py

```
@@ -432,13 +432,23 @@ def get_available_without_course(self, user):
         else:  # TA
             return qs.editable(user)
 
+    def all_archived(self, user):
+        """
+        Make and return the qs to staff users only.
+        """
+        qs = self.get_queryset(include_archived=True).filter(archived=True)
+        if user.is_staff:
+            return qs
+        else:
+            return []
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)